WordPress/src/main/java/org/wordpress/android/ui/publicize/PublicizeButtonPrefsFragment.java

```diff
@@ -322,6 +328,11 @@ public void run() {
     private void setPreferencesFromSiteSettings() {
         assignPrefListeners(false);
         try {
+            if (!mSiteSettings.isSharingModuleEnabled()) {
```

Comment:
Do you think it'd be better to set the visibility of `mSharingDisabledNotification` and `mSharingSettingsWrapper` a bit more dynamically? So for example, if we decided to add pull to refresh to this page and the user enables sharing after the page is loaded (so the notification is visible) this wouldn't actually hide the notification and show the settings.

It wouldn't actually do anything right now, but will future proof the code a bit. What do you think?

Reply:
I can totally do that. I think I had that implemented at one point in time 👍